### ISSUE TITLE
merge to original repo

### ISSUE THREAD

**Issue by kopach** [ivanzusko/mostly-adequate-guide-uk#1]
How about to merge what you have till now to original repo? In that case it will be much easier to find Ukrainian version for readers. Also, someone else could contribute to Ukrainian translations easier, without forking your fork. And if you decide to continue contributing - you can always sync your fork after all.

**Comment by ivanzusko** [comment]
@kopach the original repo doesn't contain translations, only link to translations (link to **Ukrainian** translation is present on the main repo). But, as soon as this translation will be complete - this repo will be moved to the **MostlyAdequate**

**Comment by kopach** [comment]
@ivanzusko, yes you're right. my bad, sorry. indeed there is a link to Ukrainian translation from main repo (but only from english version). Probably I was trying to find Ukrainian translation being on some other then English translation :) e.g: https://github.com/MostlyAdequate/mostly-adequate-guide-it/blob/master/TRANSLATIONS-it.md
Anyway, would be great to update those non-English files as well. What do you think?

**Comment by ivanzusko** [comment]
@kopach I think it would be nice 👍  And I think you can easily do it by yourself 😉  
this week I'm very busy, so if you will not make **PR**s with changes - I'll do them on weekends! Just let me know, so we will not do the same work in separate PRs 😄 

Cheers!

**Comment by kopach** [comment]
@ivanzusko, sorry, wont mage to do this at least till next weekend.
